web/src/beta/components/Popover/index.tsx

```diff
@@ -79,7 +79,9 @@ export const Content = React.forwardRef<
   if (!isMounted) return null;
 
   return (
-    <FloatingPortal>
+    <FloatingPortal
+      root={context.refs.reference as React.MutableRefObject<HTMLDivElement>}
+      preserveTabOrder>
```

Comment:
perserveTabOrder is by default true, so you don't need to add it 👍🏻

Reply:
Removed